FirebaseAuth/Sources/Swift/Backend/RPC/MultiFactor/SignIn/StartMFASignInResponse.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 
 @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
 struct StartMFASignInResponse: AuthRPCResponse {
-  var responseInfo: AuthProtoStartMFAPhoneResponseInfo?
+  let responseInfo: AuthProtoStartMFAPhoneResponseInfo
```

Comment:
This diff should be a no-op in practice because the initializer will throw if it could not unwrap the value being assigned to `responseInfo`.